use_cases/CI_pipeline/gulpfile.js

```diff
@@ -1,19 +1,20 @@
 /*
- * This program and the accompanying materials are made available under the terms of the *
- * Eclipse Public License v2.0 which accompanies this distribution, and is available at  *
- * https://www.eclipse.org/legal/epl-v20.html                                            *
+ * This program and the accompanying materials are made available and may be used, at your option, under either: *
+ * * Eclipse Public License v2.0, available at https://www.eclipse.org/legal/epl-v20.html, OR *
+ * * Apache License, version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0 *
  *                                                                                       *
- * SPDX-License-Identifier: EPL-2.0                                                      *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0                                        *
  *                                                                                       *
  * Copyright Contributors to the Zowe Project.                                           *
+ *                                                                                       *
  */
 
 var cmd = require('node-cmd'),
     config = require('./config.json'),
     fs = require('fs'),
     gulp = require('gulp-help')(require('gulp')),
-    gulpSequence = require('gulp-sequence'),
-    PluginError = require('plugin-error');
+    gulpSequence = require('gulp-sequence');
+// var PluginError = require('plugin-error');
```

Comment:
Is this commented out because of an unused import? If so should it be removed? We should probably avoid commented out code in our sample scripts where possible 🙂

Reply:
👍 I agree.
Will remove in a minute.